lib/redmine/menu_manager/top_menu/projects_menu.rb

```diff
@@ -53,7 +53,7 @@ def render_projects_dropdown
   end
 
   def project_items
-    [project_index_item, project_new_item]
+    [project_index_item]
```

Comment:
Did you discuss this with Wieland whether we want to remove this item?

Reply:
Yep, that's why it is this way. It wouldn't be hard to put it back in (basically 2 or 3 reverse cherry picks), but we talked abou thaving the right side of the app-header for global items, and the left side for project-specific stuff